### DISCUSSION TITLE
Usind "and" in a condition on dates

### DISCUSSION THREAD

**Discussion by VinceT** [PostgREST/postgrest#3251] [Q&A] [answered]
Hi,
I'm trying to use "and" in a select but can't figure how to write it properly.
This is ok :
http://localhost:8100/devise?ts=gte.2021-01-01&ts=lte.2022-12-31&order=ts.desc&select=code,ts
But this (that should be equivalent) isn't :
http://localhost:8100/devise?and=(ts.gte.2021-01-01,ts.lte.2022-12-31)&order=ts.desc&select=code,ts
I tried casting using "::" but got no success. Type of field "ts" is timestamp.
Could someone point me in the right direction ?
Thanks.

**Comment by VinceT** [comment]
Typo : 
http://localhost:8100/devise?and=(ts.gte.2021-01-01,ts.lte.2022-12-31)&order=ts.desc&select=code,ts
is actually
http://localhost:8100/devise?and(ts.gte.2021-01-01,ts.lte.2022-12-31)&order=ts.desc&select=code,ts
But it doesn't work either

**Reply by wolfgangwalther** [reply]
No, `and=...` is correct.

**Comment by wolfgangwalther** [comment]
> But this (that should be equivalent) isn't:
> `http://localhost:8100/devise?and=(ts.gte.2021-01-01,ts.lte.2022-12-31)&order=ts.desc&select=code,ts`

This looks ok to me.

What result are you getting for the two queries?

**Reply by VinceT** [reply, marked as answer]
Grrr!!!
Problem of cache !
This works perfectly 
http://localhost:8100/devise?and=(ts.gte.2021-01-01,ts.lte.2022-12-31)&order=ts.desc&select=code,ts 
Thanks
